lib/src/main/kotlin/org/walletconnect/impls/MoshiPayloadAdapter.kt

```diff
@@ -186,7 +187,7 @@ class MoshiPayloadAdapter(moshi: Moshi) : Session.PayloadAdapter {
                 "to" to to,
                 "nonce" to nonce,
                 "gasPrice" to gasPrice,
-                "gasLimit" to gasLimit,
+                "gas" to gasLimit,
```

Comment:
Would be nice if we also rename the param, but not critical

Reply:
That would be breaking change and clients would need to update. Also I personally think gasLimit is clearer in the library API

Reply:
yeah, thought so. I am fine as it is, go ahead and merge it :)